packages/zoe/src/zoe.js

```diff
@@ -415,6 +415,10 @@ const makeZoe = (additionalEndowments = {}) => {
     const contractFacet = harden({
       reallocate: (offerHandles, newAllocations, sparseKeywords) => {
         assertOffersHaveInstanceHandle(offerHandles, instanceHandle);
+        assert(
```

Comment:
Please insert a TODO comment pointing to our discussion about how this is a temporary kludge to be replaced by a static test.

Reply:
Hmm, I'm happy to put a comment that we will revisit this to see if a static test might be sufficient, but I think I've shown that reallocating with zero or one offers is more like adding zero or one terms together (in other words, it's not merely a no-op, it does not make any conceptual sense, and there is no good reason to do it), than it is like adding the value 0. Saying that this is a temporary kludge seems to be ignoring this point.

Reply:
I thought I had convinced you that this was a false analogy. Since I hadn't, it is still a live controversy what the right answer is. That much more reason to insert a link to the continuing discussion!

Reply:
Oh, no, I didn't see any reply to the analogy, only the question of why static testing isn't enough. Putting a link in to the discussion sounds like a very good idea!